ndk-context/src/lib.rs

```diff
@@ -0,0 +1,87 @@
+//! Provides a stable api to rust crates for interfacing with the android platform. It is
+//! initialized by the runtime, usually [__ndk-glue__](https://crates.io/crates/ndk-glue),
+//! but could also be initialized by java or kotlin code when embedding in an existing android
+//! project.
```

Comment:
Just in case, perhaps mention here that having this crate duplicated in the crate graph is dangerous (and we should do everything in our power to never make breaking changes again).

Reply:
Dangerous in the sense that you'll get annoying errors, same as we get right now and are trying to solve by moving to a crate that's a bit more stable.

Reply:
well, I guess that's up to us to make sure we don't release breaking changes. I think it's simple enough that we can say we nailed it.